index.js

```diff
@@ -0,0 +1,225 @@
+	function hasClass(el, className) {
+	  if (el.classList) {
+  		return el.classList.contains(className);
+	  }	 else {
+	  	return !!el.className.match(new RegExp('(\\s|^)' + className + '(\\s|$)'));
+	  }	    
+	}
+
+	function addClass(el, className) {
+	  if (el.classList) {
+  		el.classList.add(className);
+	  }	    
+	  else if (!hasClass(el, className)) {
+	  	el.className += " " + className;
+	  }
+	}
+
+	function removeClass(el, className) {
+	  if (el.classList) {
+	  	el.classList.remove(className);
+	  }	else if (hasClass(el, className)) {
+	    var reg = new RegExp('(\\s|^)' + className + '(\\s|$)');
+	    el.className=el.className.replace(reg, ' ');
+	  }
+	}
```

Comment:
Shows why jQuery is useful, eh? :wink: Did you write these from scratch? If not, please mention your source!

http://advanced-js.github.io/syllabus/#instructor